BREAKING.md

```diff
@@ -1,4 +1,6 @@
 ## 0.45 Breaking changes
+- [Changes to dataStoreRuntime and dataStoreContext interfaces]
```

Comment:
These should be in 0.46 breaking changes. 0.45 is already released. Also, it's only one method that is removed, so I would call it out specifically - `documentId` removed from x and y interfaces.